server/repository/src/db_diesel/name.rs

```diff
@@ -266,13 +270,14 @@ impl<'a> NameRepository<'a> {
 
 impl Name {
     pub fn from_join(
-        (name_row, (_name_link_row, name_store_join_row, store_row), name_oms_fields): NameAndNameStoreJoin,
+        (name_row, (name_link_row, name_store_join_row, store_row), name_oms_fields): NameAndNameStoreJoin,
     ) -> Name {
         Name {
             name_row,
             name_store_join_row,
             store_row,
             properties: name_oms_fields.properties,
+            name_link_row,
```

Comment:
Exposing name_link_row on the name object, so can correctly associate name link and name in new loader